### PR TITLE
ref(ui): Cleanup audit log UI

### PR DESCRIPTION
**Pull request by evanpurkhiser**
It's just been irking me a little bit

Old busted:

image

New hotness
image

(note UTC in the 2nd screenshot is bc of env differences)

### CODE REVIEW

**Review by billyvg — COMMENTED** [review]
Can we use a `<PanelTable>` here? https://storybook.getsentry.net/branches/master/index.html?path=/story/ui-panels--panel-table

Aside, the Action and IP columns back to back look kind of awkward

**Comment by EvanPurkhiser** [comment]
I can switch to `PanelTable` which will fix the look the actions too.

File: src/sentry/static/sentry/app/views/settings/organizationAuditLog/auditLogList.jsx [hunk 1]
```diff
@@ -125,18 +85,21 @@ class AuditLogList extends React.Component {
                       <Note>{entry.note}</Note>
                     </NameContainer>
                   </UserInfo>
-                  <div>{entry.event}</div>
                   <div>
+                    <MonoDetail>{entry.event}</MonoDetail>
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/settings/organizationAuditLog/auditLogList.jsx:89]: I wouldn't right align or monospace this column. The data isn't numeric values, and doesn't benefit much from monospacing.

Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/views/settings/organizationAuditLog/auditLogList.jsx:89]: I think the monospace makes sense since it's a key

Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/views/settings/organizationAuditLog/auditLogList.jsx:89]: But I agree it should be left aligned.

**Review by BYK — APPROVED** [review]
The new version is already much better. We can always address the nits later. Approving.

**Comment by EvanPurkhiser** [comment]
Spoke with @BYK, let's just land this since it's a good improvement and iterate later

**Comment by BYK** [comment]
Go home size-limit bot, you're drunk

**Pull request merged**